tasks/main.yml

```diff
@@ -43,6 +43,8 @@
   run_once: True
   always_run: True
 
+  # TODO: When there is any need to change this task, consider refactoring and
+  # using the versions provided in local facts.
```

Comment:
Nope, this would be wrong. Local facts are based on information from remote hosts; Ansible Controller does not have any facts available, and we cannot really do anything about this (unless Ansible Controller is managed by DebOps, but that isn't a given). I would remove this comment to not mislead future contributors.